### DISCUSSION TITLE
Need a way to add test variables in pytest html report.

### DISCUSSION THREAD

**Discussion by prasham28** [pytest-dev/pytest-html#711] [Q&A]
I am having tests as:
def test_func_name():
    var1 = some_val
    var2 = some_val1
 
all the tests have these same variables with different values
need a way to display values of var1 and var2 in seperate column of report for each test.

**Comment by BeyondEvil** [comment]
You can do it by adding header/column: https://pytest-html.readthedocs.io/en/latest/user_guide.html#modifying-the-results-table

You just need to figure out how to get the values you need from the test tho. That's not something this plugin can do. I think you can find that info from one of the pytest hooks.
